docs/reference/runtimes/python/python-reference.md

```diff
@@ -54,20 +58,30 @@ ADD ./my-function.yaml /etc/nuclio/config/processor/processor.yaml
 CMD [ "processor" ]
 ```
 
-### Notes
+<a id="function-configuration"></a>
+## Function configuration
+
+Your function-configuration file (for example, `my-function.yaml` for the [example Dockerfile](#dockerfile)) must include the name of your handler function and Python runtime. For more information, see the [function-configuration reference](/docs/reference/function-configuration/function-configuration-reference.md). For example:
```

Comment:
Cosmetics: use bold formatting instead of code for the name of the configuration file &mdash; `**my-function.yaml**` (as in my original review comment) &mdash; per our convention for file and directory names; we use code formatting for file names only when we're referring to part of a code that includes a file name (like the reference to the same configuration file file in the "Dockerfile" section).